bench/oracle-gas-model/page-template.html

```
@@ -0,0 +1,1035 @@
+    <label for="d-layer">Oracle contract</label>
+    <select id="d-layer">
+      <option value="bespoke">Bespoke — the custom fact store</option>
+      <option value="eas">EAS — Ethereum Attestation Service</option>
+    </select>
+    <div class="hint" id="layer-hint"></div>
+  </div>
+
+  <div class="dial">
+    <label>Token count</label>
+    <div class="seg" id="d-tokens-seg">
+      <button data-v="100">100</button><button data-v="1000">1,000</button><button data-v="100000">100,000</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)